python_sdk/infrahub_sdk/ctl/constants.py

```diff
@@ -0,0 +1,73 @@
+PROTOCOLS_TEMPLATE = """#
+# Generated by "infrahubctl protocols"
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Protocol, runtime_checkable, Optional
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Optional, Protocol, runtime_checkable
```

Ruff complains about sorted imports